crates/flox-rust-sdk/src/models/publish.rs

```diff
@@ -1234,4 +1262,19 @@ mod tests {
 
         assert_eq!(snapshot_actual, snapshot_actual);
     }
+
+    #[tokio::test]
+    async fn resolves_indirect_ref_to_git_https() {
+        let indirect_ref = FlakeRef::from_url("flake:flox", PARSER_UTIL_BIN_PATH).unwrap();
+        let (flox, _temp_dir_handle) = flox_instance();
+        // Need to expose the custom registry to `parser-util` via environment variable
+        flox.nix::<NixCommandLine>(vec![]).export_config_env_vars();
```

Comment:
I think i made a comment for this in runix already, but I'd prefer we move this into flox, e.g. by implementing that on the `Flox` struct.

Reply:
Comment addressed over there